src/services/utilServices/Sanitizer.ts

```diff
@@ -25,6 +25,9 @@ DOMPurify.addHook("uponSanitizeElement", (node, data) => {
     // Adapted from https://github.com/cure53/DOMPurify/blob/e0970d88053c1c564b6ccd633b4af7e7d9a10375/src/purify.js#L719-L736
     DOMPurify.removed.push({ element: node })
     try {
+      if (!node.parentNode) {
+        throw new Error()
```

Comment:
previous our dependancies has `jsdom`, which is a javascript library, so we didnt have the type for `node.parentNode`. But which this PR I've included `types/jsdom`, which the typescript version of that library. So now we know the type of `node.parentNode` and it can be null. So the line `node.parentNode.removeChild(node)` now causes a compile error. I could do `node.parentNode?.removeChild(node)` but that won't encounter a runtime error like before, so to be consistent I throw an error if it is null.

Reply:
sure, could you include this in the error? some meaningful string would be helpful

Reply:
okay :+1: